### PR TITLE
Remove links page, place links in their desired pages

### PR DESCRIPTION
**Pull request by krichard410**
This PR covers the second point from #768 

Specifically, 
Remove learn-relevant-links or merge with another document. It does get some traffic, but is largely orphaned. We should consider moving the links like the 4 papers from the getting started page into this page, and then linking to it from the getting started page, i.e. "Polkadot has published four different papers, each for a specific audience. They're listed and described in the Polkadot Papers section of our Relevant Links page."

Note, that I've left out Substrate and Kusama links as I think they should be placed on their separate sites.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -51,6 +51,20 @@ network, please head over to our [Kusama Guide][kusama guide].
...
+- [Polkadot Explorer](https://polkadot.js.org/apps/#/explorer) - Browser for the Polkadot network;
+  can be used for Alexander, Kusama, or your own private network.
+- [Polkascan](http://polkascan.io/) \- Real-time multi-chain data for Polkadot Relay Chain and
+  Parity Substrate chains.
+- [Subscan.io](https://subscan.io) - Explorer for Substrate based chains.
+- [Polkadot Overview](https://youtu.be/lIghiCmHz0U) - Dr. Gavin Wood presents an overview of
+  Polkadot. (Video)
+- [Polkadot Overview](https://techcrunch.com/video/fireside-chat-with-jutta-steiner-parity-technologies/) -
+  Dr. Jutta Steiner presents Polkadot. (Video)
+  - [Polkadot & Substrate Overview](https://www.youtube.com/watch?v=0IoUZdDi5Is&feature=youtu.be) -
```
Comment (Swader) [inline docs/getting-started.md]:
```suggestion
- [Polkadot & Substrate Overview](https://www.youtube.com/watch?v=0IoUZdDi5Is&feature=youtu.be) -
```

File: docs/learn-governance.md [hunk 2]
```diff
@@ -377,3 +377,9 @@ same time. Voting on one does not prevent a user from voting on the other.
 
 - [Initial Governance Description](https://github.com/paritytech/polkadot/wiki/Governance)
 - [Democracy Pallet](https://github.com/paritytech/substrate/tree/master/frame/democracy/src)
+- [Governance Demo](https://www.youtube.com/watch?v=VsZuDJMmVPY&feature=youtu.be&t=24734) - Dr.
+  Gavin Wood presents the initial governance structure for Polkadot. (Video)
+- [Governance on Polkadot](https://www.crowdcast.io/e/governance-on-polkadot--) - A webinar
+  explaining how governance works in Polkadot and Kusama.
+  - [Governance on Polkadot](https://www.crowdcast.io/e/governance-on-polkadot--) - An explanation
```
Comment (Swader) [inline docs/learn-governance.md]:
```suggestion
- [Governance on Polkadot](https://www.crowdcast.io/e/governance-on-polkadot--) - An explanation
```

File: docs/learn-polkadot-host.md [hunk 3]
```diff
@@ -39,5 +39,8 @@ parts in grey are stable and can not change without an explicit hard fork.
 
 - [Polkadot Host Protocol Specification](https://github.com/w3f/polkadot-spec) - Incubator for the
   Polkadot Host spec, including tests.
+  - [ChainSafe's Go PH](https://github.com/ChainSafeSystems/go-pre) is a 25-person development team
```
Comment (Swader) [inline docs/learn-polkadot-host.md]:
```suggestion
- [ChainSafe's Go PH](https://github.com/ChainSafeSystems/go-pre) is a 25-person development team
```

**Review by Swader — CHANGES_REQUESTED** [review]
You comitted the polkadot-wiki file, please `git rm` it. This happens because when you're testing the PDF on your local branch, it will get pulled into new branches you create if you don't clear it, and since it's only in gitignore in the pdf branch it is not getting auto ignored.

**Comment by Swader** [comment]
I'm afraid this might infect our history with the binary file. @lsaether do you know if merging this would pollute history with 30MB extra? Should @krichard410 re-submit the PR from scratch without this file?

**Comment by lsaether** [comment]
That's a good question, I assume we're only polluting GitHub's git server and not the user's git client unless the user checks out the commit that included the file. _I could be totally wrong about this_

**Comment by laboon** [comment]
It will pollute the git history with the binary file so we should get rid of it.

Solution is to rebase against master and squash all commits - this will remove any superfluous files.

```
git checkout master
git pull
git checkout -
git rebase -i master
# change all commits except the top to start with "s" (for squash)
git push -f
```

Let me know if you have any issues!

**Comment by laboon** [comment]
I just did it myself - I initially forgot that you had to force-push (although that's not quite as worrisome on a branch as master)

Should be good to go with no binary in any commit

**Comment by Swader** [comment]
Thanks Bill!

**Pull request merged**